### ISSUE TITLE
Allow project updates to be ordered by actual dates of events

### ISSUE THREAD

**Issue by punchagan** [akvo/akvo-rsr#3057]
![image](https://user-images.githubusercontent.com/315678/34517406-e6943750-f0a0-11e7-99f3-1aec77d12f6c.png)

Updates for a project were created post-facto, and not in the same order in which they occurred.  It would be nice to have these updates ordered by the actual date of these events. But currently we do not support ordering project updates.

**Comment by punchagan** [comment]
There is no UI in 'My RSR' where we actually show all the updates associated with a project. At least not until #3045 is done. We could then easily have added the re-ordering interface. 

Instead, we could add another timestamp field on the ProjectUpdate model that captures the actual event occurrence date.  If these dates are set correctly, we would have the updates ordered according to the dates of the occurrence of their corresponding events.  

![image](https://user-images.githubusercontent.com/315678/34517868-d200b762-f0a2-11e7-91c2-d5d5a422dd89.png)

Users could go back and edit these dates from the Update editing interface, too.
